mantidproject/mantid
# Removed the LoadAscii Custom Dialog, it was out of date with the alg

**Description of work.**

The custom dialog was out of date with the algorithm, and was not useful 
because:
1. it only selected the , seperator if the file extension was csv (and the algorithm has automatic seperator detection by default anyway)
1. If used via Load the custom dialog was skipped anyway

So I've removed it as the default one is fine, and stays up to date with the alg.

**Report to:** nobody

**To test:**

1. Just use the LoadAscii algorithm from the default dialog and check you are happy

Fixes #22872. 

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
